substrate/client/executor/polkavm/Cargo.toml

```diff
@@ -17,7 +17,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 log = { workspace = true }
-polkavm = { workspace = true }
+polkavm = { version = "0.16.0", default-features = false }
```

Comment:
Stick to using the workspace version please. Just update the version there. It is not used anywhere else.

Reply:
OK I will do that eventually. Thanks for the tip.